content.json

```diff
@@ -1561,6 +1561,75 @@
           }
         ]
       },
+      {
+        "title": "iOS Development YouTube Channels",
+        "slug": "youtube",
+        "description": "YouTube channels cover a wide variety of topics like tutorials, investigations, contracting and soft skills just to mention a few. Here you'll find all the channels that cover such topics.",
+        "sites": [
+          {
+            "title": "Brian Advent",
+            "author": "Brian Advent",
+            "site_url": "https://www.youtube.com/channel/UCysEngjfeIYapEER9K8aikw",
+            "feed_url": "https://www.youtube.com/feeds/videos.xml?channel_id=UCysEngjfeIYapEER9K8aikw"
+          },
+          {
+            "title": "ForrestKnight",
+            "author": "Forrest Knight",
+            "site_url": "https://www.youtube.com/channel/UC2WHjPDvbE6O328n17ZGcfg",
```

Comment:
I'd say that this channel is only barely related to iOS development? I agree he is an iOS developer, but I don't really feel this channel is *about* iOS development.